.circleci/config.yml

```diff
@@ -124,7 +124,7 @@ jobs:
 
       - restore_cache:
           keys:
-            - client-dependencies-{{ checksum "client/package-lock.json" }}-{{ checksum "client-patch-checksums.txt" }}-v2
+            - client-dependencies-{{ checksum "client/package-lock.json" }}-{{ checksum "client-patch-checksums.txt" }}-v3
```

Comment:
Can you remind me why this needed to roll all the keys forward? That still necessary?

Reply:
Leftovers when trying to debug failing CI tests